jkube-kit/generator/api/src/main/java/org/eclipse/jkube/generator/api/support/BaseGenerator.java

```diff
@@ -227,6 +231,14 @@ protected void addLatestTagIfSnapshot(BuildConfiguration.BuildConfigurationBuild
         }
     }
 
+    protected void addTagsFromConfig(BuildConfiguration.BuildConfigurationBuilder buildConfigurationBuilder) {
```

Comment:
Tags should be consistent with the requirements from https://docs.docker.com/engine/reference/commandline/tag/
`A tag name must be valid ASCII and may contain lowercase and uppercase letters, digits, underscores, periods and dashes. A tag name may not start with a period or a dash and may contain a maximum of 128 characters.`

Reply:
Tag validation is currently by docker APIServer while processing tag request submitted by jkube. 

Is it necessary to do validation again on our side?